static/sass/partials/_header.scss

```diff
@@ -1,19 +1,129 @@
 /* SASS file for header. */
 header {
+	/*Sitename and menu.*/
```

Comment:
Indentation issues on this file. Please fix.

Reply:
Done.